Source/WebCore/dom/Element.cpp

```diff
@@ -265,6 +265,9 @@ Element::~Element()
         if (auto* map = elementRareData()->attributeStyleMap())
             map->clearElement();
     }
+
+    if (langAttrKnownToMatchDocumentElement())
```

Comment:
By our naming convention, this would probably be renamed to hasLangAttrKnownToMatchDocumentEleement.